emgapianns/management/commands/import_analysis.py

```diff
@@ -159,14 +160,11 @@ def handle(self, *args, **options):
         logger.info("The upload of the run/assembly {} finished successfully.".format(self.accession))
 
     def __find_existing_result_dir(self, secondary_study_accession, run_accession, version):
-        """
-
-        :param run_accession:
-        :param version:
-        :return:
+        """Find the results folder 
         """
         logging.info("Finding result directory...")
-        directory = os.path.join(self.rootpath, '2019')
+        # FIXME: remove hardcoded value.
+        directory = os.path.join(self.rootpath, '2022')
```

Comment:
My understanding of the hardcoded `'2022'` is that it just speeds up the search by looking in the `2022/` folder before failing upwards to look in other years. If so... maybe we could at least change this to `str(datetime.datetime.now().year)` in case we don't get back to improving this by December?

Reply:
Yes, that is it. But, I really want to address that one. I'm going to leave it like that and will add a JIRA ticket to address it